wiki/Tournaments/osu!_at_UCSD/TGEX_2022/en.md

```diff
@@ -0,0 +1,27 @@
+---
+tags:
+  - LAN
+  - osu! at UCSD
+  - osu! UCSD
+  - San Diego
+  - TGEX
+  - TGEX2022
+  - UCSD
+---
+
+# Triton Gaming Expo 2022
+
+![Triton Gaming Expo 2022 announcement banner](https://cdn.discordapp.com/attachments/228229738181820417/971166619948699678/TGEX.png)
+
+[osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) is hosting an osu! LAN tournament at **Triton Gaming Expo 2022** (***TGEX 2022***) in collaboration with [Triton Gaming](https://discord.gg/tritongaming "Triton Gaming Discord server"). Attendance and tournament participation is open to everybody. The exact schedule and details for the tournament have not been released yet.
```

Comment:
```suggestion
**Triton Gaming Expo 2022** (***TGEX 2022***) is an osu! LAN tournament hosted by [osu! at UCSD](/wiki/Community/Organisations/osu!_at_UCSD) in collaboration with [Triton Gaming](https://discord.gg/tritongaming). Attendance and tournament participation is open to everybody. The exact schedule and details for the tournament have not been released yet.
```

Reply:
the lan doesnt have any real title, it's just a thing they're doing at tgex

Reply:
then why is the article called TGEX 2022 🤔 

maybe "TGEX 2022 osu! LAN tournament"?

Reply:
feels natural to me to call these types of events by their parents' names, e.g. Zotcade at UCI. if it matters to make it more precise u can change it tho.

Reply:
well, probably ok for now